### PR TITLE
Display plain "Inbound" in peer details

### PR DESCRIPTION
**Pull request by jonatack**
Alternative version to #201.

### CODE REVIEW

File: doc/release-notes-162-163.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  connected through: IPv4, IPv6, or Onion, along with the ability to display two
+  potential future additions, I2P and CJDNS. (#162)
+
+- In the Peers window, the NodeId and Node/Service column headers are renamed to
+  Peer Id and Address, respectively. (#162)
+
+- In the Peers window, the Direction field in the peer details area is replaced
+  by a Connection Type field that displays both the direction and the type of
+  peer connection. For more info, place the cursor over the field to see the
+  detailed tooltip. (#163)
```
Comment (MarcoFalke) [inline doc/release-notes-162-163.md:15]: Not sure if we need this. Most people don't read the release notes and making them longer with gui changes, which are (should aim to be) self-explanatory in most cases, doesn't make them more attractive. No strong opinion, though.

**Review by MarcoFalke — APPROVED** [review]
review ACK ff8afeb22296460ab4e04eaab48ce0f89f8036bf

Thanks

File: doc/release-notes-162-163.md (hunk 1 above)
Comment (jonatack) [reply doc/release-notes-162-163.md:15]: Yes, I wasn't sure either. I also didn't check if the GUI PR numbers work with the scripts.

Comment (jonatack) [reply doc/release-notes-162-163.md]: Dropped the release notes, can add them later on if anyone wants.

**Comment by hebasto** [comment]
> Alternative version to #201.

Concept ACK, but #201 still [preferred](https://github.com/bitcoin-core/gui/pull/201#issuecomment-770271780) to me, tbh.

**Comment by MarcoFalke** [comment]
ACK 506e6585a54818d0613a067f91c0bac2f308a48c

**Comment by jonatack** [comment]
`-netinfo` on master is now aligned with this change:

Screenshot from 2021-02-05 17-14-34

**Review by jonasschnelli — APPROVED** [review]
utACK 506e6585a54818d0613a067f91c0bac2f308a48c

**Review by lehuuhieu7777 — COMMENTED** [review]
@ldenman

**Pull request merged**